dev/doc/debugging.md

```diff
@@ -59,6 +59,25 @@ Debugging from Caml debugger
      from the debugger. If this happens, unset the variable, re-start Emacs, and 
      run the debugger again.
 
+Debugging with ocamldebug from the command line
+===============================================
+
+1. (for Cygwin/Windows) Pass the -no-custom option to the configure script before building Coq.
+2. Run "make byte"
+3. (for Cygwin/Windows) Add the full pathname of the directory .../kernel/byterun to your bash PATH.
+   Alternatively, copy the file kernel/byterun/dllcoqrun.dll to a directory that is in the PATH.  (The
+   CAML_LD_LIBRARY_PATH mechanism described at the end of INSTALL isn't working.)
+4. Run "dev/ocamldebug-coq bin/coqtop.byte.exe"
+5. Enter "source db" to load printers
+6. See the ocamldebug manual for more information.  A few points:
+   - use "break @ Printer 501" to set a breakpoint on line 501 in the Printer module (printer.ml)
+   - "backtrace" to see the call stack
+   - "step" goes into called functions; "next" skips over them
+   - "list" shows the code just before and after the current stack frame
+   - "print <var>" to see the value of a variable
+7. If you change the Coq code, you may only need a "make byte", which is much quicker.  "make"
+   will recompile all of the *.v files, which takes much loner.
```

Comment:
Because this is a Markdown file, you should put `*.v` between backticks (otherwise the `*` is interpreted as starting some italic test).
You could also change quotes for backticks around `make` and `make byte` and possibly add some around paths.
Finally, note that `make byte` is not included in `make` anymore, so not only is it quicker, it is also compulsory to get an up-to-date `coqtop.byte`.